socorro/external/postgresql/products.py

```diff
@@ -14,11 +18,131 @@
 logger = logging.getLogger("webapi")
 
 
-class Products(PostgreSQLBase):
+class SmartDate(object):
+
+    def clean(self, value):
+        if any(itertools.imap(value.startswith, ('>=', '<='))):
+            operator_ = value[:2]
```

Comment:
No, but it's a common builtin module that we might import. However, I think it's ugly like that. I'll change it to `op`.